content/en/docs/concepts/services-networking/service.md

```diff
@@ -184,6 +184,38 @@ An ExternalName Service is a special case of Service that does not have
 selectors and uses DNS names instead. For more information, see the
 [ExternalName](#externalname) section later in this document.
 
+### Endpoint Slices
+{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+
+Endpoint Slices are a new API resource designed to provide a more scalable
```

Comment:
It's best to avoid placing text like this at a particular point in history: at some point, that _new_ will be out of date.

Reply:
Good call, thanks!